build.gradle

```diff
@@ -36,7 +36,7 @@ deploy {
 def deployArtifact = deploy.targets.roborio.artifacts.frcCpp
 
 // Set this to true to enable desktop support.
-def includeDesktopSupport = false
+def includeDesktopSupport = true
```

Comment:
Can run the command Change Desktop Build Confguration or similar name to set to false, or manually edit